movement/filtering.py

```diff
@@ -1,13 +1,19 @@
 """Filter and interpolate tracks in ``movement`` datasets."""
 
+import logging
 import warnings
 from typing import Literal
 
 import xarray as xr
 from scipy import signal
 
-from movement.utils.logging import log_to_attrs, logger
+feat / bidirectional - displacement - filter
```

Comment:
@demoncoder-crypto the CI failure is coming from this line. Not sure how that crept in, but you have to remove it.

Reply:
I suspect that your branch name was somehow left in the file after you resolved the merge.